ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -31,34 +31,74 @@
 ### ISIS Interfaces Summary
 
 | Interface | ISIS Instance | ISIS Metric | Interface Mode |
-| -------- | -------- | -------- | -------- |
+| --------- | ------------- | ----------- | -------------- |
 {# ethernet interfaces #}
 {# ------- #}
-{%      if ethernet_interfaces is defined and ethernet_interfaces is not none %}
-{%          for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
-{%              if ethernet_interfaces[ethernet_interface].isis_enable is defined and ethernet_interfaces[ethernet_interface].isis_enable is not none %}
-| {{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric | default('-') }} | {% if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is defined and ethernet_interfaces[ethernet_interface].isis_network_point_to_point is not none %} point-to-point {% elif ethernet_interfaces[ethernet_interface].isis_passive is defined and ethernet_interfaces[ethernet_interface].isis_passive is not none %} passive {% else %} - {% endif %}|
+{%      for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
+{%          if ethernet_interfaces[ethernet_interface].isis_enable is arista.avd.defined %}
+{%              set row_isis_instance = ethernet_interfaces[ethernet_interface].isis_enable %}
+{%              set row_isis_metric = ethernet_interfaces[ethernet_interface].isis_metric | default('-') %}
```

Comment:
Should be arista.avd.default('-')

Reply:
My bad, just copy-pasted without looking. Fixed now.